docs/_sass/elements.scss

```diff
@@ -122,11 +122,6 @@ li:has(> a[href*="/guides/tutorials"]) {
   margin-left: -2rem;
 }
 
-li:has(> a[href*="/extensions"]) {
-  list-style-type: none;
-  margin-left: -2rem;
-}
```

Comment:
This will affect other place where we have a[href*="/extensions"], have you tested it is not affecting other places?

Reply:
I checked through the UI across the site and found out that it was not affecting any other place where a[href*="/extensions"] is used.